Library/ViewModels/ProjectPageViewModel.swift

```diff
@@ -588,6 +588,8 @@ private func fetchProject(projectOrParam: Either<Project, Param>, shouldPrefix:
             |> Project.lens.personalization.backing .~ projectWithBacking.backing
             |> Project.lens.personalization.isBacking .~ true
             |> Project.lens.extendedProjectProperties .~ projectWithBacking.project.extendedProjectProperties
+            // FIXME: Seems like in the `fetchBacking` call we nil out the chosen currency set by `fetchProject` b/c the query for backing doesn't have `me { chosenCurrency }`, so its' being included here.
```

Comment:
Can we create a ticket and link here to address this?

Reply:
Yea that might be case but because GQL has performance costs for each nested fragment, unless we see many cases where we need the `me { chosenCurrency }` from the `fetchBacking` call, we should probably just get the value internally, especially because already have it and its' not a value that changes often.. I guess I'll change this to "INFO" not fixme. 👍🏿